src/network/tarcap/packets_handlers/dag_packets_handler.cpp

```diff
@@ -34,10 +31,6 @@ void DagPacketsHandler::process(const dev::RLP &packet_rlp, const PacketData &pa
                                 const std::shared_ptr<TaraxaPeer> &peer) {
   if (packet_data.type_ == PriorityQueuePacketType::kPqNewBlockPacket) {
     processNewBlockPacket(packet_rlp, packet_data, peer);
```

Comment:
no need to call this function, probably just copy everything here as it is for other handlers

Reply:
Done. Also removed some additional unused code related to the change.